VictoriaLogs: When no data is written for a long time, a panic occurs when writing data

### Describe the bug

When no data is written for a long time (more than `-retentionPeriod`), a panic occurs when writing data

### To Reproduce

1. When no data is written for a long time (more than `-retentionPeriod`)
2. Directory `/storage/partitions` is empty.
3. Write data.



### Version

master

### Logs

```go
panic: runtime error: invalid memory address or nil pointer dereference

goroutine 124454 [running]:
github.com/VictoriaMetrics/VictoriaMetrics/lib/httpserver.handlerWrapper.func1()
	github.com/VictoriaMetrics/VictoriaMetrics/lib/httpserver/httpserver.go:235 +0x65
panic({0x892480, 0xcd5d30})
	runtime/panic.go:890 +0x263
github.com/VictoriaMetrics/VictoriaMetrics/lib/logstorage.(*partitionWrapper).decRef(0x414df1?)
	github.com/VictoriaMetrics/VictoriaMetrics/lib/logstorage/storage.go:167 +0x32
github.com/VictoriaMetrics/VictoriaMetrics/lib/logstorage.(*Storage).MustAddRows(0xc0000d9ea0, 0xc000000000)
	github.com/VictoriaMetrics/VictoriaMetrics/lib/logstorage/storage.go:404 +0x745
github.com/VictoriaMetrics/VictoriaMetrics/app/vlstorage.MustAddRows(...)
	github.com/VictoriaMetrics/VictoriaMetrics/app/vlstorage/main.go:79
github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/elasticsearch.RequestHandler({0xc00002cc7a, 0x6}, {0xaa8c20, 0xc0000d9f40}, 0xc000032b00)
	github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/elasticsearch/elasticsearch.go:99 +0x62f
github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert.RequestHandler({0xaa8c20, 0xc0000d9f40}, 0xc000032b00)
	github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/main.go:35 +0x185
main.requestHandler({0xaa8c20?, 0xc0000d9f40}, 0xc000032b00)
	github.com/VictoriaMetrics/VictoriaMetrics/app/victoria-logs/main.go:87 +0x2da
github.com/VictoriaMetrics/VictoriaMetrics/lib/httpserver.handlerWrapper(0xc00029a3b0, {0xaa8c20?, 0xc0000d9f40}, 0xc000032b00, 0xa3a300)
	github.com/VictoriaMetrics/VictoriaMetrics/lib/httpserver/httpserver.go:344 +0xd8d
github.com/VictoriaMetrics/VictoriaMetrics/lib/httpserver.gzipHandler.func1({0xaa8c20?, 0xc0000d9f40?}, 0xc000430540?)
	github.com/VictoriaMetrics/VictoriaMetrics/lib/httpserver/httpserver.go:196 +0x37
net/http.HandlerFunc.ServeHTTP(0xcd7ce0?, {0xaa8c20?, 0xc0000d9f40?}, 0x4?)
	net/http/server.go:2122 +0x2f
github.com/klauspost/compress/gzhttp.NewWrapper.func1.1({0xaa8b30, 0xc000430540}, 0xc000142ad0?)
	github.com/klauspost/compress@v1.16.5/gzhttp/compress.go:478 +0x35a
net/http.HandlerFunc.ServeHTTP(0x0?, {0xaa8b30?, 0xc000430540?}, 0x65ed94?)
	net/http/server.go:2122 +0x2f
net/http.serverHandler.ServeHTTP({0xaa7938?}, {0xaa8b30, 0xc000430540}, 0xc000032b00)
	net/http/server.go:2936 +0x316
net/http.(*conn).serve(0xc0001b6630, {0xaa8ed8, 0xc000276690})
	net/http/server.go:1995 +0x612
created by net/http.(*Server).Serve
	net/http/server.go:3089 +0x5ed
```

### Screenshots

_No response_

### Used command-line flags

```shell
/victoria-logs-prod --retentionPeriod=3d --storageDataPath=/storage --envflag.enable=true --envflag.prefix=VM_ --loggerFormat=json
```

### Additional information

![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/15684156/ace6e289-3977-4b10-8b23-d3f212f06959)

So i think we need to add `nil` check for `ptwHot.pt`.

--- a/lib/logstorage/storage.go
+++ b/lib/logstorage/storage.go
@@ -395,7 +395,7 @@ func (s *Storage) MustAddRows(lr *LogRows) {
 	}
 	s.partitionsLock.Unlock()
 
-	if ptwHot != nil {
+	if ptwHot != nil && ptwHot.pt != nil {
 		if ptwHot.canAddAllRows(lr) {
 			ptwHot.pt.mustAddRows(lr)
 			ptwHot.decRef()

